cpp/src/examples/image_classifier/mnist/mnist_handler.cc

```diff
@@ -7,9 +7,9 @@ void MnistHandler::Postprocess(
   for (const auto& kv : idx_to_req_id) {
     try {
       auto response = (*response_batch)[kv.second];
-      response->SetResponse(200, "data_tpye",
-                            torchserve::PayloadType::kDATA_TYPE_BYTES,
-                            torch::pickle_save(torch::argmax(data[kv.first])));
+      response->SetResponse(
+          200, "data_tpye", torchserve::PayloadType::kDATA_TYPE_BYTES,
+          torch::pickle_save(torch::argmax(data[kv.first].softmax(0))));
```

Comment:
this would still return probabilities of all the classes instead of just the highest class, is that intended?

Reply:
this implementation is trying to match https://github.com/pytorch/serve/blob/master/examples/image_classifier/mnist/mnist_handler.py#L36